derive_action_signing/module_data/deposit.py

```diff
@@ -15,11 +18,20 @@ class DepositModuleData(ModuleData):
     decimals: int
 
     def to_abi_encoded(self):
+        amount_decimal = Decimal(self.amount)
+        scaled_amount = int(amount_decimal.scaleb(self.decimals))
```

Comment:
@joshpwrk please confirm that decimals is necessary as a lot of the trades for example use e18

Reply:
can you explain a bit more? For deposits we need decimals since WBTC is e8 and USDC is e6.